tasmota/include/tasmota_template.h

```diff
@@ -198,6 +198,9 @@ enum UserSelectablePins {
   GPIO_ADE7953_RST,                    // ADE7953 Reset
   GPIO_NRG_MBS_TX, GPIO_NRG_MBS_RX,    // Generic Energy Modbus device
   GPIO_ADE7953_CS,                     // ADE7953 SPI Chip Select
+#ifdef ESP32
```

Comment:
I understand the idea but let's keep the list in sync between both architrctures to make documenting easier.

So pls remove the test for ESP32 here

Reply:
performed